src/styles/blocks/catalog.scss

```
@@ -17,9 +17,15 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: $transition-time;
 
     &:not(:nth-child(4n)) {
       margin-right: 48px;
     }
+
+    &:hover {
+      transform: scale(1.2);
+      color: #34568b;
```

if you already added transition variable, you can add color)
```suggestion
      color: $c-someColor;
```